src/EditorFeatures/CSharpTest/GenerateFromMembers/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersTests.cs

```diff
@@ -59,15 +56,17 @@ public Program(int i, string s)
         this.i = i;
         this.s = s;
     }
-}", title: string.Format(FeaturesResources.Add_parameters_to_0, "Program(int)"));
```

Comment:
hrmm.. testing the title is deliberate.  is it not possible to have this?

Reply:
It's possible, but there were two other tests that did only that (`TestSmartTag*`) so I was being lazy with the rest.

I will be un-lazy.